CODEOWNERS

```
@@ -1,3 +1,3 @@
 #ECCN:Open Source
-#GUSINFO:Heroku Front End Dept (FE Infra & Arch),Heroku Front-End Arch
+#GUSINFO:Heroku FE Infra & Arch,Heroku Front-End Arch
```

```suggestion
#GUSINFO: Heroku FE Dev Tools,Heroku CLI & Plugins
```
Shouldn't this be assigned to the dev tooling team in GUS?